README.md

```diff
@@ -7,19 +8,18 @@ DAGFiNN is a conversational conference assistant that can be made available for
 
 # Features
 
-
-  * Provide recommendations across multiple domains, have basic conversational capabilities, and personalization integrated into a single system.
-    - Currently, recommenders are built for point-of-interest (POI) and conference content
-  * Support multiple user interfaces (UI)
-    - A Web chat and a Furhat robot with an optional external screen
-    - Use of rich responses customized to each user interface to effectively convey information.
-  * Utilize multiple input modalities
-    - Furhat is a social robot with a microphone, speaker, and camera, which makes it possible to recognize users that interacted with the robot previously.
-
+- Provide recommendations across multiple domains, have basic conversational capabilities, and personalization integrated into a single system.
```

Comment:
Nit: I do not understand why this list has been modified as it follows our [guidelines](https://github.com/iai-group/guidelines/tree/main/markdown#lists).

Reply:
Changed back